server/models/v1/show.js

```diff
@@ -2,26 +2,26 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const showSchema = new Schema({
-  status: String,
+  status: String, //what are the possible values?
```

Comment:
Right now I am assigning active to all new shows.  I was thinking we can do Active and Cancelled/Deleted at a minimum and add other later as necessary.  I can set this up as an enumeration.

Reply:
@jberns an enumeration would be great! I'll set "active" as the default.